### ISSUE TITLE
Insightful error message

### ISSUE THREAD

**Issue by amirouche** [biwascheme/biwascheme#75]
When something is wrong with my code, I'm not sure how to debug since the only clue i have is this strange error message on web console:

```
TypeError: BiwaScheme.Port.current_error.puts is not a function
```

I am using 0.6.5 in firefox nightly

**Comment by yhara** [comment]
Could you paste any program which prints that error?

**Comment by amirouche** [comment]
For instance this program in the browser does raise the error specified above:

```scheme
(+ 1 (abc 2))
```

The error is reported at biwascheme.js:8695:5

**Comment by yhara** [comment]
Thanks, I've reproduced the bug. Let me think how to fix it.

**Comment by yhara** [comment]
@amirouche This should be fixed on the git master. I'll release a new version in a week, probably with some more fixes (eg. #74)

**Comment by amirouche** [comment]
Thx!

**Comment by yhara** [comment]
Released 0.6.6.
https://github.com/biwascheme/biwascheme/releases/tag/0.6.6

**Comment by amirouche** [comment]
It's fixed :+1:
